hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/HandleWorkflow.java

```diff
@@ -76,6 +78,7 @@ public class HandleWorkflow {
     private final TransactionChecker checker;
     private final ServiceScopeLookup serviceScopeLookup;
     private final ConfigProvider configProvider;
+    private final Time time;
```

Comment:
`Time` has already been moved into a separate module (`com.swirlds.base`). So far, the classes in this module were picked with platform in mind. We may want to revisit that. But I think we will need a common base with classes like `Time`, units, etc.

I introduced `Time` in platform because I probably do not understand the JDK classes. 🙂 What I usually need in a test is to set the clock to a specific time, call method A, advance the clock by x ms, and call method B. I don't know how this can be achieved with the provided implementations because they are either constant or based on the system clock. Providing an implementation of `Clock` looks much more complex than defining your own `Time`. Using `InstantSource` is more lightweight. But it requires creating an `Instant` every time you want to get the current millis. I.e. the JDK classes appear overly complicated and inefficient. I probably do not understand them.